src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/ObTableOperationType.java

```diff
@@ -26,7 +26,10 @@ public enum ObTableOperationType {
     INSERT_OR_UPDATE(4), // INSERT or UPDATE, columns not in arguments will remain unchanged
     REPLACE(5), // DELETE & INSERT, columns not in arguments will change to default value
     INCREMENT(6), // the column must be can be cast to long. if exist increase, else  insert
-    APPEND(7);// append column value
+    APPEND(7),// append column value
+    SCAN(8),// scan value
+    TTL(9),// expire value
```

Comment:
What is the type of TTL operation type

Reply:
Here ObTableOperationType is same as definition of observer。TTL is an special operation in observer。